vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -108,7 +109,8 @@ func NewClusterStateFeeder(config *rest.Config, clusterState *model.ClusterState
 	kubeClient := kube_client.NewForConfigOrDie(config)
 	podLister, oomObserver := NewPodListerAndOOMObserver(kubeClient, namespace)
 	factory := informers.NewSharedInformerFactoryWithOptions(kubeClient, defaultResyncPeriod, informers.WithNamespace(namespace))
-	controllerFetcher := controllerfetcher.NewControllerFetcher(config, kubeClient, factory)
+	crdCache := crdcache.NewCrdCache(config, make(chan struct{}), defaultResyncPeriod)
```

Comment:
stop channel should provided via function params.

Reply:
I don't know what you mean. I deleted stop chan param and follow the rules in other package to create stop chan in crdcache.

Reply:
I was thinking about having stop channel as an input parameter. This would be helpful if cluster feeder was recreated many times during recommender life. 
However, I looked at the recommender code and it didn't seem like a use case. Moreover other informers are also not propagating stop channels.
Deleting stop channel seems ok.